backend/events/search-indexer/src/main/java/com/meemaw/search/indexer/KafkaUtils.java

```diff
@@ -6,8 +6,8 @@
 @UtilityClass
 public class KafkaUtils {
 
-  private final String BOOTSTRAP_SERVERS = "BOOTSTRAP_SERVERS";
-  private final String DEFAULT_BOOTSTRAP_SERVERS = "localhost:9092";
+  private final String BOOTSTRAP_SERVERS = "KAFKA_BOOTSTRAP_SERVERS";
+  private final String DEFAULT_BOOTSTRAP_SERVERS = "PLAINTEXT://kafka:9092";
```

Comment:
Can you check why env variable does not work here? @Meemaw

Reply:
Maybe because you named it dofferently in manifet.